### PR TITLE
feat(geo): add manual configuration steps

### PR DESCRIPTION
**Pull request by TreTuna**
_Description of changes:_
Add documentation around manual setup for the Geo category to use existing Amazon Location Service resources.

image

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by renebrandel — CHANGES_REQUESTED** [review]
Can you move this to an individual section like this one: https://docs.amplify.aws/lib/graphqlapi/existing-resources/q/platform/android

**Comment by TreTuna** [comment]
> Can you move this to an individual section like this one: https://docs.amplify.aws/lib/graphqlapi/existing-resources/q/platform/android

Will do!

File: docs/lib/geo/fragments/existing-resources.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+You can also use Amplify Geo with your existing Amazon Location Service resources if you'd like, there's just some manual setup that you need to do.
+
+### Authorization permissions
```
Comment (renebrandel) [inline docs/lib/geo/fragments/existing-resources.md:3]: Use H2

**Review by renebrandel — APPROVED** [review]

File: docs/lib/geo/fragments/existing-resources.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+You can also use Amplify Geo with your existing Amazon Location Service resources if you'd like, there's just some manual setup that you need to do.
+
+## Authorization permissions
+
+If you set up your Cognito resources manually, the roles will need to be given permission to access the map and place indices.
+
+There are two roles created by Cognito: an `Auth_Role` that grants signed-in-user-level access and an `Unauth_Role` that allows unauthenticated access to resources. Attach the corresponding policies to each role for proper access. Replace ```{account-id}``` and ```{enter Map/PlaceIndex name}``` with the correct ids and resource names.
```
Comment (Amplifiyer) [inline docs/lib/geo/fragments/existing-resources.md:7]: Region needs to be made placeholder as well.

File: docs/lib/geo/fragments/existing-resources.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Add the following policy for both the `Auth_Role` and `Unauth_Role`:
```
Comment (Amplifiyer) [inline docs/lib/geo/fragments/existing-resources.md:9]: I'd say we shouldn't tell user to add to both as an instruction, rather add following policies for the appropriate roles (Auth and/or Unauth)

File: docs/lib/geo/fragments/js/existing-resources.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+Amplify.configure({
+  Auth: {
+    identityPoolId: 'XX-XXXX-X:XXXXXXXX-XXXX-1234-abcd-1234567890ab', // REQUIRED - Amazon Cognito Identity Pool ID
+    region: 'XX-XXXX-X', // REQUIRED - Amazon Cognito Region
+    userPoolId: 'XX-XXXX-X_abcd1234', // OPTIONAL - Amazon Cognito User Pool ID for authenticated user access
+    userPoolWebClientId: 'XX-XXXX-X_abcd1234', // OPTIONAL - Amazon Cognito Web Client ID for authenticated user access
+  },
+  Geo: {
+    AmazonLocationService: {
```
Comment (Amplifiyer) [inline docs/lib/geo/fragments/js/existing-resources.md:16]: Does this work? I thought we needed `geo` and `amazon_location_services` as the keys

File: docs/lib/geo/fragments/existing-resources.md (hunk 2 above)
Comment (TreTuna) [reply docs/lib/geo/fragments/existing-resources.md:7]: Good catch!

File: docs/lib/geo/fragments/js/existing-resources.md (hunk 4 above)
Comment (TreTuna) [reply docs/lib/geo/fragments/js/existing-resources.md:16]: Spoke on this offline. It's primarily the `geo/Geo` part right now that effects this. Will update to `geo`, but leave `AmazonLocationServices` for now. Will be having a conversation with other JS devs on if this needs to stay consistent with other categories as far as the casing goes and we will update as/if needed.

File: docs/lib/geo/fragments/existing-resources.md (hunk 2 above)
Comment (TreTuna) [reply docs/lib/geo/fragments/existing-resources.md]: Good catch, updated!

File: docs/lib/geo/fragments/existing-resources.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+You can also use Amplify Geo with your existing Amazon Location Service resources if you'd like, there's just some manual setup that you need to do.
+
+## Authorization permissions
+
+To use your existing Amazon Location Service resources, i.e. maps and place indices, with Amplify Geo, you need to ensure your role has the right authorization permissions through Cognito. If you set up your Cognito resources in any other way other than through the CLI or Admin UI, the roles will need to be given permission to access the map and place indices.
```
[the remark this answers is not in the document]
Comment (TreTuna) [reply docs/lib/geo/fragments/existing-resources.md]: Updated

**Review by hdworld11 — APPROVED** [review]

**Pull request merged**